src/osgview/GUIOSGBuilder.cpp

```diff
@@ -379,7 +379,13 @@ GUIOSGBuilder::buildMovable(const MSVehicleType& type) {
     if (myCars.find(osgFile) == myCars.end()) {
         myCars[osgFile] = osgDB::readNodeFile(osgFile);
         if (myCars[osgFile] == 0) {
-            WRITE_ERROR("Could not load '" + osgFile + "'.");
+            WRITE_ERROR("Could not load '" + osgFile + "'. The model is replaced by a box shape.");
```

Comment:
actually the error message is now wrong it is not a box anymore

Reply:
Was corrected in commit https://github.com/eclipse/sumo/commit/1a8d867cbe00b3991ff7e213848a1e72958452a2